playbooks/ansible/roles/samba.install/files/bootstrap-centos.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
```

Comment:
Going forward installation of packages might not be the only step during bootstrap process. It can very well include others like custom configuration of repositories, export of environment variables(in a non-standard installation) or literally any other action for which usage of ansible modules can become very limited. Therefore I think its good to have a separate collective bootstrap script rather than bombarding under ansible shell or command modules.

Reply:
Since at the moment we only install packages which can be handled by ansible yum module, for the sake of consistency, we should use the playbook to install these packages. We can add additional steps later as needed.

Reply:
This is the introduction of playbook itself and I think its wise and harmless to have such a bootstrap script in this very early stage anticipating any future additions. Comparatively with CentOS we are more likely to add something extra down the line sooner or later.